modules/memfault/memfault_flash_coredump_storage.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2021 Memfault Inc.
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <pm_config.h>
```

Comment:
You should not need this here as it is inluded by `<storage/flash_map.h> -> <flash_map_pm.h> -> <pm_config.h>`